### PR TITLE
debug script for people to test openmm

### LINKED ISSUE #313: [docs] troubleshooting your installation
Things like:
- checking PYTHONPATH issues
- checking version problems
- are you running the latest versions of openfe/gufe?

### LINKED ISSUE #843: [docs] Showcase notebook failing to render
```
/home/docs/checkouts/readthedocs.org/user_builds/openfe/checkouts/841/docs/tutorials/showcase_notebook.nblink:1159: WARNING: Literal block expected; none found.
```

### LINKED ISSUE #844: Unpin rdkit on mypy
Opening up this PR so we remember to do this.

@mikemhenry I believe you put the pin in, do you know if there's an issue upstream to work out why this duplicate self is happening?

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
* [x] Added a ``news`` entry

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

Fixes #313 
Fixes #843
Fixes #844

### CODE REVIEW

File: devtools/debug_openmm.sh
Comment (IAlibay) [inline devtools/debug_openmm.sh:1]: @mikemhenry this looks good - would it be worth point to it in / augmenting the docs section where you already added the debug info for all this? (nvidia-smi calls etc..)

File: devtools/debug_openmm.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+echo "Run this script with you conda env activated"
```
[the remark this answers is not in the document]
Comment (mikemhenry) [reply devtools/debug_openmm.sh:3]: Thanks!

[the remark this answers is not in the document]
Comment (mattwthompson) [reply devtools/debug_openmm.sh:3]: No, thank you! Since I stole this https://github.com/openforcefield/status/blob/main/devtools/support/debug.sh

**Comment by mikemhenry** [comment]
If you want to test this, you will need to use this funky url: `bash -c "$(curl -Ls https://raw.githubusercontent.com/OpenFreeEnergy/openfe/bebb85e530a07a200cff044b2375cbe5f21e22d7/devtools/debug_openmm.sh)" | tee -a debug.log`

**Comment by mikemhenry** [comment]
@IAlibay and @richardjgowers should I add a note on where to post the bug report or email?

Let me know if you think anything needs extra explanation as well!

**Comment by mikemhenry** [comment]
I can't seem to find the warning in the build log https://readthedocs.org/projects/openfe/builds/24221623/

**Comment by mikemhenry** [comment]
found it!
```
/home/docs/checkouts/readthedocs.org/user_builds/openfe/checkouts/733/docs/tutorials/showcase_notebook.nblink:1159: WARNING: Literal block expected; none found.
```

**Comment by mikemhenry** [comment]
this won't fix it but I want to push up that change to see if any new issues pop up

**Comment by mikemhenry** [comment]
Okay mypy is happy and so is RTD :tada: 

Going to merge this in and then fix the other CI issues in a different PR.

**Pull request merged**